symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/impl/RequeueEventException.java

```diff
@@ -0,0 +1,18 @@
+package com.symphony.bdk.core.service.datafeed.impl;
+
+import com.symphony.bdk.core.service.datafeed.EventException;
+import com.symphony.bdk.core.service.datafeed.RealTimeEventListener;
+import com.symphony.bdk.gen.api.model.V4Event;
+
+import org.apiguardian.api.API;
+
+/**
+ * Internal exception to convey the message that ack id should not be updated thus re-queuing events.
+ */
+@API(status = API.Status.INTERNAL)
+public class RequeueEventException extends RuntimeException {
+  public RequeueEventException(V4Event event, RealTimeEventListener listener, EventException e) {
+    super(String.format("Listener %s failed to process event %s, events will be re-queued",
```

Comment:
Let's log the event type as well, that would help for debugging

Reply:
+1